app/AiPhoto/page.tsx

```diff
@@ -29,23 +34,26 @@ export default function AiPhotoPage() {
         setImage(imageURL);
         setIsLoading(false);
         setPrompt('');
+        await pb.collection('photos').create({ "image": imageURL, "user": storedUser });
     };
 
     return (
         <div className='h-screen bg-gray-800 '>
             {isLoading ? (
-                <div className='relative top-4 flex flex-col gap-y-4 self-center h-screen'>
-                    <h1 className='text-4xl text-center mt-6 ml-36 font-bold text-gray-50'>Ai Generation Tool</h1>
-                    <Img className='bg-gray-500 relative top-14 left-12 flex self-center shadow-2xl rounded-2xl'
-                         width='720' height='720' src={require('./img.png')} alt='image'/>
+                <div className="min-h-screen  flex items-center justify-center bg-secondary">
```

Comment:
Usually you should extract this loading component into a separate file

Reply:
the reason being pretty code? in my head it doesn t make sense to have a whole new component just for 5-6 lines of code